microsoft/iot-cardboard-js
# OAT: Add auto layout on import flow

### Summary of changes 🔍 
Added a dialog that shows a progress indicator for the import as well as a confirmation screen at the end. When the dialog is closed, the graph gets laid out.
![image](https://user-images.githubusercontent.com/57726991/218827343-c295c921-0629-41c5-ab38-dec2876a0fec.png)

![image](https://user-images.githubusercontent.com/57726991/218827383-1b81bfbf-65a5-4cf7-991c-06e88ae5d999.png)

![Graph Auto layout on import](https://user-images.githubusercontent.com/57726991/218831472-bc91b10e-9c44-489d-bfae-00e2c5cac8e5.gif)

### Testing 🧪
- Import a single file --> see the dialog flash quickly to complete
- Import a large ontology --> see the dialog stays up for a while and then gives a confirmation
